src/hal/src/queue/capability.rs

```diff
@@ -50,7 +50,9 @@ impl Capability for Compute {
 impl Capability for Transfer {
     fn supported_by(qt: QueueType) -> bool {
         match qt {
-            _ => true
+            QueueType::General |
+            QueueType::Transfer => true,
```

Comment:
Compute and Graphics queues supports transfer too.

Reply:
Somehow slipped my mind.

Reply:
Fixed